client/src/components/_common/Paginator/Paginator.module.scss

```diff
@@ -0,0 +1,29 @@
+.root {
```

Comment:
__Proposal__ (does not block approval)

Add a comment explaining why this extra nesting exists. Much of Portal CSS nests unnecessarily, but this nesting is intentional.

> Why not just comment the <code>!important</code> we had before?!
>
> — Developer (Who is Tired of Wes)

Now, that I have had the Internet to download and test my original idea, I see that `!important` and a comment  would have been simpler. Sorry. You could go back to that.

<details><summary>Complete Answer</summary>

\
Here's why I _both_ did **not** approve important _and_ did **not** think a comment would be necessary:

1. Adding `!important` would require any external overrides of Paginator to _also_ use `!important`, thus playing CSS specificity's fight-fire-with-fire game, but with [`!important`, which is a death star](https://confluence.tacc.utexas.edu/display/UP/UI+-+CSS+Style+Guide+-+Specificity?preview=/101384277/104693936/css-specificity-wars.png).
1. I expected that a more specific selector would have let the code itself explain what was being overwritten, e.g.
    ```
    .existing-global-problem-class .new-component-good-class {
      prop-to-overwrite: new-value;
    }
    ```

    But, what you have done is not what I expected—the code I had expected would **not** have worked. I had thought that a `.btn .current` would be adequate to overwrite Bootstrap, __but__:
    1. That requires syntax y'all don't know (and I want to avoid), `:global(.btn) .current`.
    2. That selector is not specific enough, because Bootstrap is not the culprit—`.workbench-content .btn…` is.
    3. We actually need _either_ `:global(.workbench-content) .btn.current` _or_ [FP-545](https://jira.tacc.utexas.edu/browse/FP-545)+[FP-546](https://jira.tacc.utexas.edu/browse/FP-546).

</details>

Reply:
I'll leave a general comment, but to be quite honest I can't figure out the *exact* reason why it's necessary. There's also weird shenanigans like in dev mode when hot reload occurs it *appears* to be not necessary because the changed file overrides bootstrap's css styles when the style is applied on top of it, but doesn't work in the actual order that the css is applied.